.gitignore

```diff
@@ -53,6 +53,7 @@ share/python-wheels/
 .installed.cfg
 *.egg
 build/
+_build/
```

Comment:
repeated build line?

Reply:
No - the second line has a leading underscore so corresponds to a different directory (not sure if you were getting confused with *, but _ is just a literal here)